pkgs/by-name/lz/lz4json/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, lz4
+, pkg-config
+}:
+
+let
+  name = "lz4json";
+  version = "c44c51005c505de2636cc1e59cde764490de7632";  
+  sha256 = "sha256-rLjJ7qy7Tx0htW1VxrfCCqVbC6jNCr9H2vdDAfosxCA=";
+in
+(stdenv.mkDerivation {
```

Comment:
You can remove `let ...  in`
since variables declared within this is used at most once each.

```nix
(stdenv.mkDerivation { /* package conifguration */ })
```
should be replaced to
```nix
stdenv.mkDerivation { /* package configuration */ }
```